### DISCUSSION TITLE
Building my own device

### DISCUSSION THREAD

**Discussion by n3gwg** [WLAN-Pi/feedback#67] [General Inquiries] [answered]
WLAN Pi users/developers,

I am interested in building my own device and am wondering if STL files for the cases are available such that I could try to 3D Print them?

I have a Raspberry Pi 4 and a Raspberry Pi CM4 available for this project. I already have cables that provide POE to a device and am not concerned about a battery either, though I'd like to use the AX210 or COMFAST 953AX.

What hardware platform is recommended to me for my build? I care about using the latest image and getting updates most easily too. I would use the RPi4 and get the LCD, but maybe using the AX210 internally is better or worse (impact of external antennas?).

I am listening for everyone's ideas.

73,
Stuart, N3GWG

**Comment by joshschmelzle** [comment, marked as answer]
@n3gwg Nick has the Raspberry Pi 4 models available here. https://www.printables.com/model/265668-wlan-pi-r4-fascia-essentials

**Reply by n3gwg** [reply]
I must greatly appreciate your provisioning this link in addition to your work supporting the project. I shall work on getting this printed in the near future.

I am not sure if my WLAN PI is going to be based on a CM4 device (currently unused) with a carrier board that I just ordered or if I will use a Raspberry Pi 4 (8GB RAM) I have around. I am leaning in the direction of getting another Raspberry Pi 4 with 2GB of RAM as I imagine 8GB of RAM is overkill for what WLAN PI requires.

I am also now active in the WLAN slack channel as well.

**Reply by joshschmelzle** [reply]
I like the 8 GB RAM versions for development work (as some of my workflows are pretty RAM heavy), but otherwise the 2 GB should be OK for most things.
